src/components/controls/selects/SearchSelect/types.ts

```diff
@@ -1,8 +1,12 @@
 import { ReactNode } from "react"
 
 export interface BaseOption {
+  /** @deprecated since 7.12 */
```

Comment:
What do you think about something like this? I think it's good to always provide a "way out" when deprecating something.

```suggestion
  /** @deprecated since v7.12.0. `key` is not needed when using `value`, since value have to be unique. Use `value` instead! */
```